doc/archive/roadmap.md

```diff
@@ -0,0 +1,61 @@
+> _These sections have been archived, which previously existed compositely as
```

Comment:
Please remove this file and the reference to it.

Reply:
Done. Maybe make a merge commit with this PRs correct title.